functions-and-scope.js

```diff
@@ -48,6 +81,28 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+/*
+/!*-------------------CODE------------------*!/
+const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+
+function averageGrade(cijfers) {
+    let count=0;
+    let sum=0;
+
+    for (let i = 0; i < cijfers.length; i++) {
+        count++;
```

Comment:
count ++ om het totaal aantal waardes te berekenen van de array is overbodig. Je kan namelijk simpelweg cijfers.length gebruiken om het aantal entries in de array te bepalen.